releasenotesv10.md

```diff
@@ -0,0 +1,68 @@
+v10 is the largest semver major release of pg-boss in years. The API changes included below are ordered by significance.
+
+## Database changes
+PostgreSQL 12 is now the minimum supported version. If you upgrade and run `start()`, the database will automatically be upgraded. However, this release requires rebuilding almost all of the job table indexes, which may require a bit of downtime depending on the size of your queues. If this is a concern, you may extract the migration script via `getMigrationPlans()` and run it against a backup to get an estimate on downtime.
+
+If the standard auto-migration isn't desired, consider alternatives, such as running a new v10 schema side by side of a v9 schema until the v9 queues are drained. 
+
+## API changes
+
+* MAJOR: **Job retries are now opt-out instead of opt-in.** The default `retryLimit` is now 2 retries. This will cause an issue for any job handlers that aren't idempotent. Consider setting `retryLimit=0` on these queues if needed.
```

Comment:
I think this is not a good default. Configure retries when necessary seems to be a better option.